src/io/output_time.cc

```diff
@@ -212,6 +256,9 @@ void OutputTime::write_time_frame()
 			// Remember the last time of writing to output stream
 			write_time = time;
 			current_step++;
+            
+            // unset validity of the output mesh after data are written at the current write time
+            is_output_mesh_valid_ = false;
```

Comment:
I don't understand. You have:

transport output : null ouput mesh -> make it, do fields output
reactions output: mesh exists reuse it, output other fields
call write_time_frame, output everything and delete output_mesh

So the point is that I see no save in resolution of invalid and null mesh since both mean just make it from scratch.